README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/CS450-ECE461/fall2016-group2-apiserver.svg?branch=master)](https://travis-ci.org/CS450-ECE461/fall2016-group2-apiserver)
+[![Build Status](https://travis-ci.org/CS450-ECE461/fall2016-group2-apiserver.svg?branch=development)](https://travis-ci.org/CS450-ECE461/fall2016-group2-apiserver)
 # fall2016-group2-apiserver
 
```

Comment:
I need for coveralls to run on the dev branch to get the badge. If I added the badge for issue20, we wouldn't see the actual coverage of the dev branch with new merges.